Assets/Scripts/Claire table/FindCoconuts.cs

```diff
@@ -4,28 +4,30 @@
 
 public class FindCoconuts : MonoBehaviour
 {
-    //used to detect an object before a collision
-
     public delegate void CoconutAction();
     public static event CoconutAction CoconutAcquired;
 
+    //used to detect an object before a collision
+
     RaycastHit hit;
     public float distance;
     public static bool hasKey;
     private bool collectedPrize;
+    private int layerMask;
 
     private void Start()
     {
         collectedPrize = false;
         hasKey = false;
+        layerMask = 1 << 9;
```

Comment:
I'm okay with not changing this, but you can grab layers by their name which makes it a bit more robust if someone reorganizes layers.